### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by VladimirUman**

### CODE REVIEW

File: src/services/authentication.js [hunk 1]
```diff
@@ -12,28 +13,34 @@ const parseJwt = (token) => {
     }
 };
 
-function getLocalAccessToken() {
+async function getLocalAccessToken() {
     let accessToken = sessionStorage.getItem('accessToken');
 
     if (!accessToken) {
+        accessTokenSubject.next(null);
```
Comment (MaxLogvyniukFive) [inline src/services/authentication.js]: What this code do?

**Review by MaxLogvyniukFive — CHANGES_REQUESTED** [review]

File: src/services/authentication.js (hunk 1 above)
Comment (VladimirUman) [reply src/services/authentication.js]: update observable token object

**Review by MaxLogvyniukFive — APPROVED** [review]

File: src/api/index.js [hunk 2]
```diff
@@ -31,13 +33,12 @@ api.interceptors.response.use(
         const originalConfig = err.config;
 
         if (err.response) {
-            const authErrorsStatusCodes = [419, 401];
             // Access Token was expired
-            if (authErrorsStatusCodes.includes(err.response.status) && !originalConfig._retry) {
+            if (err.response.status === 419 && !originalConfig._retry) {
```
Comment (MaxLogvyniukFive) [inline src/api/index.js:37]: Put 419 in variable

File: src/services/authentication.js [hunk 3]
```diff
@@ -12,33 +11,60 @@ const parseJwt = (token) => {
...
     if (!accessToken) {
+        currentUserSubject.next(null);
+
         return null;
     }
 
     const tokenData = parseJwt(accessToken);
 
     if (!tokenData) {
+        currentUserSubject.next(null);
```
Comment (MaxLogvyniukFive) [inline src/services/authentication.js:47]: What does this action do?

**Pull request merged**